oraclebmc/util.py

```diff
@@ -99,3 +100,14 @@ def __bool__(self):
         return self.truthy
     # PY2 Compatibility
     __nonzero__ = __bool__
+
+def initkwargs(fn):
+    @functools.wraps(fn)
+    def init(self, **kwargs):
+        fn(self)
+        for kw, value in kwargs.items():
+            if kw not in self.swagger_types:
+                raise TypeError("%s doesn't have atribute %s" %(self.__class__, kw))
```

Comment:
nit: typo in "atribute" and please use `str.format()`.
There may some lingering usage of the traditional style, but those will eventually be replaced.